hackathon_site/dashboard/frontend/src/slices/hardware/cartSlice.ts

```diff
@@ -49,6 +94,20 @@ const cartSlice = createSlice({
             cartAdapter.updateOne(state, payload);
         },
     },
+    extraReducers: (builder) => {
+        builder.addCase(submitOrder.pending, (state) => {
+            state.isLoading = true;
+            state.error = null;
+        });
+        builder.addCase(submitOrder.fulfilled, (state) => {
+            state.isLoading = false;
+            state.error = "Submitted";
+        });
+        builder.addCase(submitOrder.rejected, (state) => {
+            state.isLoading = false;
+            state.error = "Submitted order has been rejected";
```

Comment:
Can you set this to the reject value message for now? I think that's at `payload.error.message`, but could be wrong. We might do that in some other slices.